backend/src/main/java/de/neuefische/backend/controller/APIController.java

```diff
@@ -29,4 +30,9 @@ public NasaPicture getPictureOfTheDay (){
     public NasaPicture getRandomPicture(){
         return apiService.getRandomPicture();
     }
+
```

Comment:
In der Zeile 25 wurde ich die Methoden Zeichen "()" neben Methodennamen schreiben, also: getPictureOfTheDay()... das ist Konvention.

Bei Deinen Requests @Get.. holst Du mehrere Sachen oder? Ich würde Dir empfehlen, falls es so ist, die Sachen in Plural zu schreiben: z.B.:
getPicturesOfTheDay()
getRandomPictures()
getArchives()